### ISSUE TITLE
Subcommand and Python API to visualize particles updates given a saved model

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/ShapeWorks#569]
If the user saved the particle system over iterations, e.g., run on a server, this subcommand/API should provide the user to visualize saved particle updates to trouble shoot correspondence optimization.

**Comment by akenmorris** [comment]
Does this mean that we launch Studio?

**Comment by sheryjoe** [comment]
I think so. It would be redirecting Studio to get particle updates from saved checkpoints for visualization. I think implementing this outside Studio would entail duplicate viz functionality. Other thoughts?

**Comment by akenmorris** [comment]
Wait, how does a remote server fit into this?  We need network communication?

**Comment by sheryjoe** [comment]
do you mean cloud? or if the user ran his optimization on a server? For this issue, we assume that the input is an (accessible) path where shapeworks optimize results were saved.

**Comment by akenmorris** [comment]
I see. So you mean you want to monitor a folder for changes being made by another shapeworks process, potentially on a different machine.

**Comment by sheryjoe** [comment]
No. Assume that the user has already optimized a shape model while keeping checkpoints and saving them to the shapeworks output folder (using the keep_checkpoints xml tag). This command just visualizes particles saved at these checkpoints, i.e. retrospectively visualizing particles updates, rather than life updates in Studio. Does this make sense?

**Comment by akenmorris** [comment]
Oh, I see, yes. Cool. I’ve been thinking about adding a history slider to let you go back and forth through the optimization.

**Comment by sheryjoe** [comment]
Yep. Exactly. This will also help a user who needs a hefty machine to optimize the model using batch processing (command line optimize) but also need a way to troubleshoot the correspondence optimization process offline.
